dist/main.css

```diff
@@ -0,0 +1,94 @@
+.jumbotron {
+  padding-top: 48px;
+  padding-bottom: 48px;
+}
+
+body {
+  margin: 0px;
+}
+
+#app {
+  background-color: #DCDCDC;
+  text-align: center;
+  padding: 0;
+  display: block;
+  margin: 0;
+}
+
+label {
```

Comment:
Do you understand the repercussions of styling elements vs classes? 
If you style an element it will affect every element even if you did not want it to. 
Try to use classes where you do not want such global solutions

Reply:
keep in mind specificity of css; applying a style to an element will change all those specific elements UNLESS a class or ID is applied to the element which will override the element style, maybe this was implied?

Reply:
@Darren443 has a better answer